sdk/storage/Azure.Storage.Files.Shares/src/ShareFileClient.cs

```diff
@@ -1899,20 +1973,29 @@ private void SetNameFieldsIfNull()
         /// Optional <see cref="ShareFileRequestConditions"/> to add conditions on
         /// the download of the file.
         /// </param>
+        /// <param name="cancellationToken">
+        /// Optional <see cref="CancellationToken"/> to propagate
+        /// notifications that the operation should be cancelled.
+        /// </param>
         /// <returns>
         /// Returns a stream that will download the file as the stream
         /// is read from.
         /// </returns>
         internal Stream OpenReadInteral(
             long position = 0,
             int? bufferSize = default,
-            ShareFileRequestConditions conditions = default)
+            ShareFileRequestConditions conditions = default,
+            CancellationToken cancellationToken = default)
         {
             DiagnosticScope scope = ClientDiagnostics.CreateScope($"{nameof(ShareFileClient)}.{nameof(OpenRead)}");
             try
             {
                 scope.Start();
-                return new LazyLoadingReadOnlyStream<ShareFileRequestConditions>(
+
+                // Have to use cancellationToken in body.
+                Debug.Assert(cancellationToken != null || cancellationToken == null);
```

Comment:
This can cause test issues.  Just suppress the warning for now.  Same for all the other uses.

Reply:
Fixed in next iteration.